lib/registries/index.js

```diff
@@ -36,5 +36,10 @@ module.exports = EngineAddon.extend({
                 { transformation: 'amd', as: 'immutable' },
             ],
         });
+        this.import('node_modules/unicode-byte-truncate/index.js', {
```

Comment:
I don't know if I did this right. Copy pasta. There was something complaining about being broken but i don't remember what it was. I did not get these working properly.